.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @lolgab @ljmf00 @andreaTP @rtfpessoa @bmbferreira @DReigada @pedrocodacy
+* @codacy/toss
 
 *.yml @h314to @paulopontesm
```

Comment:
Can you pls remove this line and leave only toss team?

Reply:
Done ✅